dice/README.md

```diff
@@ -0,0 +1,27 @@
+---------------------------
+Author Name:IanGSalmon
+**Dice.com Element**  
+This element allows the creation of an element instance for the Dice Talent Public API.
+#  
```

Comment:
@IanGSalmon Could you provide a link to the vendor's documentation in the Readme?